bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/FileInputImpl.java

```diff
@@ -69,16 +71,10 @@ public Integer getMaxItems() {
 
     @Override
     public Type getType() {
-        // if (isMultiple()) {
-        // return Type.ARRAY;
-        // } else {
-        // }
-        // file upload does not work for type string in core component, hence default it to file
-        Type superType = super.getType();
-        if (Type.STRING.equals(superType)) {
+        if (!isMultiple()) {
```

Comment:
Can you add a comment here, that type string in fileinput fieldtype is not supported in core components

Reply:
Added!